crates/astria-conductor/src/conductor.rs

```diff
@@ -152,13 +155,31 @@ impl Conductor {
             let (shutdown_tx, shutdown_rx) = oneshot::channel();
             shutdown_channels.insert(Self::DATA_AVAILABILITY, shutdown_tx);
             let block_verifier = BlockVerifier::new(sequencer_client_pool.clone());
+            let sequencer_namespace = {
```

Comment:
I believe this is problematic if `execution_commit_level == FirmOnly` (or whatever the variant is). You might not have a sequencer on the other side, but this code would still run.

The only way to fix this IMO is to not couple conductor-bringup with sequencer and instead set it through conductor's config file.

Then instead when firing up the sequencer reader you can hit the endpoint and verify that the configured sequencer namespace matches the remote one.

But that only works in cases where `! firm_only`

Reply:
Resolved: even in `firm-only` conductor needs access to sequencer.